article-api/src/main/kotlin/com/sns/article/component/article/application/ArticleCommandService.kt

```diff
@@ -55,7 +58,13 @@ class ArticleCommandService(
 
         articleRepository.delete(article)
         commentRepository.findAllByRootIdInAndRootType(listOf(articleId.id.toString()), Comment.Root.Type.ARTICLE)
-            .let { commentRepository.deleteAll(it) }
-        // FIXME reaction 을 삭제하는 방법 확인 (DB 정의가 없음)
+            .let {
+                commentRepository.deleteAll(it)
+                reactionRepository.findAllByTargetIn(it.stream()
+                    .map { a -> ReactionTarget(ReactionTargetType.COMMENT, a.id!!) }
+                    .toList())
+                    .let { reactions -> reactionRepository.deleteAll(reactions) }
```

Comment:
```suggestion
                reactionRepository.findAllByTargetIn(it.map { r -> ReactionTarget(ReactionTargetType.COMMENT, r.id!!) }
                    .let { reactions -> reactionRepository.deleteAll(reactions) }
```

kotlin 에서는 stream 으로 변환하지 않아도 사용할 수 있어요